front/tests/008-allowances.spec.ts

```diff
@@ -25,41 +19,30 @@ test.describe('Testing if all mandatory elements simulation configuration are lo
 
     await expect(scenarioPage.getTimetableList).toBeVisible();
 
-    await scenarioPage.clickBtnByName('Afficher/masquer le détail des trains');
-
-    await scenarioPage.getBtnByName(/Train Schedule 1/).hover();
-
-    await scenarioPage.clickBtnByName('Modifier');
+    await scenarioPage.getBtnByName(scenarioName).hover();
+    await page.getByTestId('edit-train').click();
 
+    // Add and check standard allowance
     await scenarioPage.openAllowancesModule();
     await expect(scenarioPage.getAllowancesModule).toBeVisible();
-
-    // Add and check standard allowance
     await scenarioPage.setStandardAllowance();
-    await scenarioPage.clickBtnByName('Modifier le train');
-
+    await page.getByTestId('submit-edit-train-schedule').click();
     expect(await scenarioPage.isAllowanceWorking()).toEqual(true);
 
-    await scenarioPage.getBtnByName(/Train Schedule 1/).hover();
-    await scenarioPage.clickBtnByName('Modifier');
-    await scenarioPage.openAllowancesModule();
-    await expect(scenarioPage.getAllowancesModule).toBeVisible();
+    await scenarioPage.getBtnByName(scenarioName).hover();
+    await page.getByTestId('edit-train').click();
 
     // Add and check engineering allowance
+    await scenarioPage.openAllowancesModule();
+    await expect(scenarioPage.getAllowancesModule).toBeVisible();
     await scenarioPage.setEngineeringAllowance();
     await scenarioPage.clickSuccessBtn();
     expect(scenarioPage.checkAllowanceEngineeringBtn());
-    await scenarioPage.clickBtnByName('Modifier le train');
+    await page.getByTestId('submit-edit-train-schedule').click();
     await scenarioPage.page.waitForSelector('.scenario-details-name');
 
     expect(await scenarioPage.isAllowanceWorking()).toEqual(true);
-  });
-
-  // ***************** Deleting train *****************
 
-  test.afterEach(async ({ page }) => {
-    const scenarioPage = new PlaywrightScenarioPage(page);
-    await scenarioPage.getBtnByName(/Train Schedule 1/).hover();
-    await scenarioPage.page.getByRole('button', { name: 'Supprimer' }).click();
+    // TODO: check if train schedule travel time is well updated (waiting for issue # 6695 to be done)
```

Comment:
```suggestion
    // TODO: add more checks to see if the allowances are taken into account  in the scenario page (waiting for issue # 6695 to be fixed)
```

Reply:
are you sure that we need more than one check?

Reply:
done